File: concrete/src/main/java/pcb/uwu/unit/base/AmpereUnit.java
package pcb.uwu.unit.base;

import pcb.uwu.units.base.ElectricCurrentUnit;

import java.util.function.Function;

public class AmpereUnit extends ElectricCurrentUnit {

	public static final AmpereUnit AMPERE = new AmpereUnit();

	private AmpereUnit() {
		super("A", "ampere", "amperes",
				Function.identity(),
				Function.identity());
	}
}

File: concrete/src/main/java/pcb/uwu/unit/base/AmpereUnit.kt
package pcb.uwu.unit.base

import pcb.uwu.utils.identity

object AmpereUnit
    : ElectricCurrentUnit(symbol = "A",
                          singularName = "ampere",
                          pluralName = "amperes",
                          translationToCanonical = identity(),
                          translationFromCanonical = identity())